core/src/rpc/rpc_context.rs

```diff
@@ -486,9 +487,33 @@ pub trait RpcContext {
 	// Methods for relating
 	// ------------------------------
 
-	async fn relate(&self, _params: Array) -> Result<impl Into<Data>, RpcError> {
-		let out: Result<Value, RpcError> = Err(RpcError::MethodNotFound);
-		out
+	async fn relate(&self, params: Array) -> Result<impl Into<Data>, RpcError> {
+		let Ok((from, kind, to, content)) = params.needs_three_or_four() else {
+			return Err(RpcError::InvalidParams);
+		};
+
+		let kind = kind.could_be_table();
```

Comment:
This should only apply to JSON right?

Reply:
I'm maintaining consistency with the existing methods which allow strings for table names